ciao-cli/examples/README.md

```diff
@@ -0,0 +1,140 @@
+# workload examples
+
+A workload can be uploaded into the ciao datastore via the ciao-cli
+command.
+
+`ciao-cli workload create -yaml my_workload.yaml`
+
+This directory includes some sample workloads that illustrate a few
+basic workload definitions. A workload consists of a yaml file which
+defines the properties and resources the workload needs, as well as a
+yaml file which contains the cloud config to use when launching the instance.
+section.
+
+## workload properties yaml
+
+The workload properties yaml file contains information which define the
+workload. This configuration will be used for each instance that is started
+with the workload ID.
+
+The first section of the file contains some basic information about the
+workload.
+
+```
+description: "My awesome container workload"
+vm_type: docker
+image_name: "ubuntu:latest"
+```
+
+`description` is just a human readable string which will be
+displayed when you list workloads with `ciao-cli workload list`.
+
+There are two types of workloads - container based, and vm based. The `vm_type`
```

Comment:
Suggest: "There are currently two types of workloads..."

And perhaps note others will be coming over time.